packages/-ember-data/tests/unit/store/adapter-interop-test.js

```diff
@@ -32,9 +32,12 @@ module('unit/store/adapter-interop - DS.Store working with a DS.Adapter', functi
   });
 
   test('Adapter can be set as a name', function(assert) {
-    store = createStore({ adapter: 'application' });
+    const env = setupStore({ adapter: 'custom-adapter' });
+    const { store, owner } = env;
+    const CustomAdapter = DS.Adapter.extend({ isCustomAdapter: true });
```

Comment:
do we need `isCustomAdapter`, makes it look as a special prop

Reply:
we don't, just trying to make it clear it's custom :D